TribalWars/src/WebApi/Controllers/TribeController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using ApplicationCore.Dtos;
+using ApplicationCore.Entities;
+using ApplicationCore.Interfaces.Services;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using ApplicationCore.Response;
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    public class TribeController : Controller
+    {
+        private readonly IMapper _mapper;
+        private readonly ITribeService _tribeService;
+
+        public TribeController(IMapper mapper, ITribeService tribeService)
+        {
+            _mapper = mapper;
+            _tribeService = tribeService;
+        }
+
+        [HttpPost("api/v1/Tribe/CreateTribe")]
+        public async Task<IActionResult> CreateTribe([FromBody] TribeDto dto)
+        {
+            Guid userId = new Guid(); // user session soon
```

Comment:
You can use "TODO" tag here, like this:
`//TODO: Read userId from session`
That way it will be highlighted in Tasks List in Visual Studio.